doc/src/comm_modify.rst

```diff
@@ -87,12 +97,26 @@ warning is printed, if this bond based estimate is larger than the
 communication cutoff used.
 
 The *cutoff/multi* option is equivalent to *cutoff*\ , but applies to
-communication mode *multi* instead. Since in this case the communication
-cutoffs are determined per atom type, a type specifier is needed and
-cutoff for one or multiple types can be extended. Also ranges of types
-using the usual asterisk notation can be given. For granular pair styles,
-the default cutoff is set to the sum of the current maximum atomic radii
-for each type.
+communication mode *multi* instead. Since the communication
+cutoffs are determined per atom collections, a collection specifier is needed and
+cutoff for one or multiple collections can be extended. Also ranges of collections
+using the usual asterisk notation can be given. 
+Collections are indexed from 0 to N-1 where N is the total number of collections.
```

Comment:
yes, the collections should be indexed from 1 to N by the user in the input script.  Internally, they can be indexed however you like.